spec.html

```diff
@@ -136,17 +137,14 @@ <h1>PerformRealmEval ( _sourceText_, _callerRealm_, _evalRealm_ )</h1>
 				1. Assert: _callerRealm_ is a Realm Record.
 				1. Assert: _evalRealm_ is a Realm Record.
 				1. Perform ? HostEnsureCanCompileStrings(_callerRealm_, _evalRealm_).
-				1. Let _inFunction_ be *false*.
```

Comment:
this form was copied from https://tc39.es/ecma262/#sec-performeval. My bad, because the flag for direct eval is not longer needed, therefore these lets declarations are not needed anymore. Good catch!